neurom/core/population.py

```diff
@@ -108,7 +112,9 @@ def neurites(self):
 
     def _load_file(self, f):
         if isinstance(f, neurom.core.morphology.Morphology):
-            return neurom.core.morphology.Morphology(f.to_morphio(), process_subtrees=self.process_subtrees)
+            return neurom.core.morphology.Morphology(
```

Comment:
Could we maybe introduce a copy() method?

Reply:
We could but it will need a `process_subtrees` parameter and will return a 'copy' with a different internal state so I'm not sure we should name it 'copy'. What about a method like the following?
```python
def use_process_subtrees(self, process_subtrees=False):
    return neurom.core.morphology.Morphology(
        self.to_morphio(), name=self.name, process_subtrees=process_subtrees
    )
```

Reply:
Why not copy the morphology and then change its `process_subtrees` attributes? `copy` should return an identical copy of the morphology.

Reply:
Yeah we can also do this indeed. I will add it.